docs/services/postgresql/add-standby-node.md

```diff
@@ -32,6 +32,84 @@ $ cchq <env> ap setup_pg_standby.yml -e standby=[standby node]
 
 
 # Promoting a hot standby to Master
+1. Begin downtime for your site: `$ commcare-cloud <env> downtime start`
+
+2. Verify that the replication is up to date
+
+```
+$ commcare-cloud <env> run-shell-command pg1,pg2 'ps -ef | grep -E "sender|receiver"'
+
+    [ pg1 ] ps -ef | grep -E "sender|receiver"
+    postgres 5295 4517 0 Jul24 ? 00:00:01 postgres: wal sender process rep 10.116.175.107(49770) streaming 0/205B598
+
+    [ pg2 ] ps -ef | grep -E "sender|receiver"
+    postgres 3821 3808 0 Jul24 ? 00:01:27 postgres: wal receiver process streaming 0/205B598
+```
+
+Output shows that master and standby are up to date (both processing the same log).
+
+3. Promote the standby
+
 ```
-$ cchq <env> ansible-paybook promote_pg_standby.yml -e standy=[standby nde]
+$ commcare-cloud <env> ansible-paybook promote_pg_standby.yml -e standy=[standby node]
+```
+
+4. In your inventory remove `hot_standby_master` and `replication_slot` variables from your standby node, and remove the node from the `pg_standby` group.
+
+5. Update your processes to point to the newly promoted server:
+
+```
```

Comment:
Nit: you should number all of these `1.` and indent all of the stuff below the number by 4 spaces to enable auto-numbering and visually linking this point with the numbered point. It makes the flow much easier to read and allows you to quickly add new points if you forget things later. 

I use `vmd` to render my markdown to see how it looks before pushing it to github. It also works with GFM, so is often a good representation of how it will be rendered later.

Reply:
Agreed on indent of 4 https://github.com/dimagi/commcare-cloud/pull/3072/commits/1fa5eb21a38dba8853c01c5ffe5209e773bf796b

I don't understand the benefit of using `1.` for everything. Jekyll will auto number even if they aren't all `1.` and making them all 1. feels like it ruins the flow of reading these in plain text  (which I think is one of the major benefits of markdown). With that said, if its a convention the team wants to follow, I don't feel strongly enough to fight it.

I tried using `vmd` and it failed when there was whitespace inside a code block that was indented so my terminal output the following:

```
...
     3. 
          Verify that the replication is up to date


     ```bash
      $ commcare-cloud

(ansible) [17:10:38] jemord:commcare-cloud git:(je/update-postgres-standby-docs*) $
```

Jekyll appears to handle that fine as tested by https://github.com/dimagi/commcare-cloud/tree/master/docs